src/transformers/modeling_utils.py

```diff
@@ -3789,7 +3789,9 @@ def from_pretrained(
             with safe_open(resolved_archive_file, framework="pt") as f:
                 metadata = f.metadata()
 
-            if metadata.get("format") == "pt":
+            if metadata is None:
+                pass
+            elif metadata.get("format") == "pt":
```

Comment:
This is the most questionable bit of this whole PR. The timm safetensor files don't have any metadata saved alongside them. To enable the cross loading, we assume if the metadata is None then it's a "pt" file, but maybe we want to do a batch conversion of the timm weights instead?